docs/API/InstrumentFormat.md

```diff
@@ -287,32 +342,188 @@ A NumericElement represents a numeric data input and has the general form of:
+           The result is stored as the element's value.
+           
+### 2.1.6 Radio
+
+A radio element represents a set of radio buttons all attributed to the same question.
+A general example is provided below:
+
+```js
+{
+    "Type": "radio",
+    "Name": REQUIRED,
+    "Description": OPTIONAL,
+    "Options": {
+        "Values" : {
+            "v0" : "label0",
+            "v1" : "label1",
+            "v2" : "label2",
+            "v9" : "label9"
+        },
+        "Order" : {
+            "0" : "v9",
+            "1" : "v2",
+            "2" : "v1",
+            "3" : "v0"
+        },
+        "Orientation" : "horizontal|vertical",
+        "RequireResponse" : boolean
+    }
+}
+```
+
+`Type`: MUST be "radio".
+
+`Name`: Required. Follows `PageElement.Name` rules.
 
+`Description`: Optional. If not specified, the
+               radio buttons will be displayed with no accompanying text.
+               If specified, they will be displayed to the left of the buttons.
+               
+`Options.Values`: KEY REQUIRED, VALUE OPTIONAL. Contains a JSON object specifying options,
+                one of which may be selected. Each key/value corresponds
+                to `<radio value="JSONKey">JSONValue</radio>` in an
+                HTML implementation. The JSONKey contains the value
+                to be saved if selected, and the JSONValue contains
+                the human friendly text to display to the user.
+                NOTE: a JSONValue is OPTIONAL because, for a large set
+                of radio button questions, the values may be defined 
+                horizontally above the set. See `Options.Orientation`.
+                
+`Options.Order`: Optional. This specified the order values are assigned to
+                 rendered radio buttons. The `Options.Values` are going to be
+                 naturally ordered when encoded to JSON, so to preserve unnatural
+                 order, we must specify the real value order in this element.
+                 See example.
+
+`Options.Orientation`: REQUIRED. Rendering them vertically will display labels.
+                       Rendering them horizontally will require the use of a
+                       `radio-labels` element for context. See Section 2.1.7.
+
+`Options.RequireReseponse`: String. If evaluated to true, an implementation should
+                       automatically add a not_answered option to the
+                       radio buttons in addition to the values specified
+                       to allow the user to explicitly not answer a question
+                       but require that some answer is entered.
+                       If false, it should not.
+                       This is done instead of simply adding the option to
+                       Values to ensure consistency with other PageElement types
+                       such as date or text.
+                       Default: true.
```

Comment:
Same comment about the name "RequireResponse" as above.. but true isn't a string, it's a boolean.